locales/pt.json

```diff
@@ -195,19 +195,19 @@
         "approve": "Aprovar",
         "cancel": "Cancelar",
         "delete": "Deletar",
-        "readMore": "Read More"
+        "readMore": "Ler mais"
     },
     "hero": {
-        "getVaccinated": "Get vaccinated with or without a voucher",
-        "helpingResidents": "Helping residents of Japan find essential healthcare"
+        "getVaccinated": "Seja vacinado com ou sem um voucher",
+        "helpingResidents": "Ajudando moradores do Japão a encontrar cuidados com a saúde essenciais"
```

Comment:
I think "cuidados de saúde primários" would be the official term here. Or if we're referring to services, then "serviços de saúde essenciais".
<img width="640" alt="Screen Shot 2021-08-15 at 11 14 54 PM" src="https://user-images.githubusercontent.com/1096046/129481451-6818f1ac-7d21-4d55-9b65-23876d517e57.png">

Reply:
Ahh, I already merged, but please feel to open a follow-up PR if there are improvements :D